sas/src/test/resources/python_test.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
```

Comment:
It was used to generate the .tsv for date_time testing, and it's been useful for generating other test output comparison files...